portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase

```diff
@@ -413,11 +413,7 @@ definition {
 
 		FormFields.selectSelectOption(
 			fieldName = "If Input",
-			selectOption = "Is not URL");
-
-		FormFields.viewTextFieldNotPresent(
```

Comment:
@gleicelisbino, shouldn't we keep this step?

Reply:
The same for email, @natocesarrego